src/test/java/com/edmunds/tools/databricks/maven/BetterAbstractMojoTestCase.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *        Taken from https://github.com/ahgittin/license-audit-maven-plugin
```

Comment:
@l0s is this adequate when it comes to bringing in source from another project?

Reply:
You might want to copy the copyright notice as well.

Reply:
Seems like a good idea.

Reply:
ok cool, took their whole license file